src/test/java/net/syscon/elite/api/resource/impl/OffenderMovementsResourceImplIntTest_rescheduleCourtHearing.java

```diff
@@ -0,0 +1,141 @@
+package net.syscon.elite.api.resource.impl;
+
+import net.syscon.elite.api.model.CourtHearing;
+import net.syscon.elite.api.model.ErrorResponse;
+import net.syscon.elite.api.model.PrisonToCourtHearing;
+import net.syscon.elite.executablespecification.steps.AuthTokenHelper;
+import org.junit.Test;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpMethod;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.in;
+import static org.springframework.http.HttpStatus.OK;
+
+public class OffenderMovementsResourceImplIntTest_rescheduleCourtHearing extends ResourceTest {
+
+    @Test
+    public void reschedule_court_hearing() {
```

Comment:
FAO @andymarke expanded the test to schedule a hearing to be amended/revised on the the hearing date/